src/pages/bridge/Issue.tsx

```diff
@@ -395,6 +395,9 @@ function Issue(props: IssueProps): JSX.Element {
                   if (parseFloat(value) > parseFloat(selectedVault?.issuableTokens?.toString() || '0')) {
```

Comment:
(Although not related to this PR and a bug that already existed before)
I suppose we need to downscale this amount to 'whole units' ie. divide it by our decimals (12). 
You can see that it's not working as expected by testing with the vaults registered on Foucoco, the entered amount is higher than 700, but the validation is not triggered.
<img width="652" alt="image" src="https://github.com/pendulum-chain/portal/assets/6690623/3c97307c-4671-4126-a8cb-f1341b33d310">

Reply:
You are right, nice catch. I fixed it.